src/Client.js

```diff
@@ -1,13 +1,15 @@
 var extend = require('util')._extend;
 var url = require('url');
 var changeCase = require('change-case');
+var deepmerge = require('deepmerge');
 
 var request = require('superagent');
 var Promise = require('bluebird');
 var ArgumentError = require('./exceptions').ArgumentError;
 var APIError = require('./exceptions').APIError;
 var defaultOptions = require('./defaultOptions');
-var goToPath = require('./utils');
+var goToPath = require('./utils').goToPath;
+var isFunction = require('./utils').isFunction;
```

Comment:
@ngonzalvez - this does not seem to be correct. unit tests work and the code (obviously) is fully of `require()` calls ... so this hound-ci message is safe to ignore?